<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QDialog" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XIA High-Speed Software Router Demo</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGraphicsView" name="graphicsView_DAG">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>29</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Current Destination XIP Address</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Packet Format</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_Packet">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_FB_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>No fallbacks</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_FB_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1 fallback</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_FB_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2 fallbacks</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_FB_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>3 fallbacks</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Number of fallbacks</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_PS_128">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>128 bytes</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PS_256">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>256 bytes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PS_512">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>512 bytes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PS_1024">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1024 bytes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Packet size</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PS_1514">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1514 bytes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PS_192">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>192 bytes</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PS_90">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>90 bytes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Real-time packet forwarding speed</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Router">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Router</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TGen">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Traffic Generator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Monitor">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Router Monitor</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_plot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>711</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
